src/plugin/systems.rs

```diff
@@ -1631,13 +1634,9 @@ mod tests {
     #[test]
     #[cfg(all(feature = "dim3", feature = "async-collider"))]
     fn async_scene_collider_initializes() {
-        use bevy::scene::scene_spawner_system;
-
         let mut app = App::new();
-        app.add_plugins(HeadlessRenderPlugin).add_systems(
-            Update,
-            init_async_scene_colliders.after(scene_spawner_system),
-        );
+        app.add_plugins(HeadlessRenderPlugin)
+            .add_systems(PostUpdate, init_async_scene_colliders);
```

Comment:
Old test was failing because the `scene_spawner_system` was moved to its own schedule that runs in between `Update` and `PostUpdate`.

Updated to match the way `init_async_scene_colliders` is added by the actual plugin.